test/built-ins/Proxy/enumerate/call-parameters.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.11
+description: >
+    [[Enumerate]] ()
+
+    8. Let trapResult be Call(trap, handler, «target»).
+---*/
+
+var x, _target, _handler;
+var target = {
+    attr: 1
+};
+var handler = {
+    enumerate: function(t) {
+        _target = t;
+        _handler = this;
+        return {next: function() { return { done:true }; } };
+    }
+};
+var p = new Proxy(target, handler);
+
+for (x in p) {}
```

Comment:
Fair enough, I have a better idea now. Add $ERROR (or something similar) inside the loop body, because it should never be executed.